pkgs/by-name/os/ossia-score/package.nix

```
@@ -42,15 +42,15 @@
 # TODO: figure out LLVM jit
 # assert lib.versionAtLeast llvm.version "15";
 
-stdenv.mkDerivation (finalAttrs: {
+clangStdenv.mkDerivation (finalAttrs: {
   pname = "ossia-score";
-  version = "3.2.3-3";
+  version = "3.2.4";
 
   src = fetchFromGitHub {
     owner = "ossia";
     repo = "score";
-    rev = "v3.2.3-3";
-    hash = "sha256-xRqsMKwuejbl+5ljYMFhQv/j1MfnFH5MGIn9rCQG/ro=";
+    rev = "v3.2.4";
```

It might be better to just do this:

```suggestion
    rev = "v${finalAttrs.version}";
```